core/domain/email_manager.py

```diff
@@ -107,13 +107,6 @@ def log_new_error(*args, **kwargs):
     EXPLORATION_ROLE_PLAYTESTER: _EDITOR_ROLE_EMAIL_HTML_RIGHTS['can_play']
 }
 
-PUBLICIZE_EXPLORATION_EMAIL_HTML_BODY = config_domain.ConfigProperty(
```

Comment:
Remember to add this to the list of deprecated config properties in the comment in config_domain.py.